models/confirmation_test.go

```diff
@@ -147,6 +147,59 @@ func Test_NewPatientPasswordResetConfirmation(t *testing.T) {
 
 }
 
+func Test_NewPatientPasswordInfoConfirmation(t *testing.T) {
```

Comment:
that makes me think I did not add one for the Type PatientPinReset. I will create a task for doing that later